ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -1738,8 +1738,8 @@ static int zend_jit_interrupt_handler_stub(dasm_State **Dst)
 	|	//EG(vm_interrupt) = 0;
 	|	MEM_STORE_8_ZTS strb, wzr, executor_globals, vm_interrupt, TMP1
 	|	//if (EG(timed_out)) {
-	|	MEM_LOAD_8_ZTS ldrb, REG0w, executor_globals, timed_out, TMP1
-	|	cbz REG0w, >1
+	|	MEM_LOAD_8_ZTS ldrb, TMP1w, executor_globals, timed_out, TMP1
+	|	cbz TMP1w, >1
```

Comment:
This REG0x -> TMP1w change + zend_jit_get_scratch_regset() change make sense, but they are not directly related to IS_SIGNED_32BIT. It's better to commit them as separate patch - "Avoid REG0 usage for interrupt checks".

Reply:
Yes. Agree. Will update. Thanks.